scripts/done-task.js

```diff
@@ -0,0 +1,40 @@
+import { CLICK_EVENT, HIDE_CLASS } from "/scripts/constants.js";
+import createButton from "/scripts/factory/createButton.js";
+
+function onTaskComplete(
+  doneButton,
+  editButton,
+  inputElement,
+  toolbar,
+  tasks,
+  id
+) {
+  doneButton.addEventListener(CLICK_EVENT, doneTask);
+
+  function doneTask() {
+    inputElement.classList.add("done");
+    const task = tasks[id];
+    task.doneButton = true;
+    doneButton.classList.add(HIDE_CLASS);
+    editButton.classList.add(HIDE_CLASS);
+
+    const now = Date.now();
+    const diffTime = now - task.date.getTime();
+    const diffHours = diffTime / (1000 * 60 * 60);
+    const diffDays = Math.ceil(diffHours / 24);
+
+    const completeTimeButtonText =
+      diffDays === 1 ? `Completed in 1 day` : `Completed in ${diffDays} days`;
+    const completeTimeBtn = createButton(
+      "complete-time",
+      completeTimeButtonText,
+      "Time to complete the task"
+    );
```

Comment:
irrelevant context in variable name.

```suggestion
    const completedText =
      diffDays === 1 ? `Completed in 1 day` : `Completed in ${diffDays} days`;
    const completeTimeBtn = createButton(
      "complete-time",
      completedText,
      "Time to complete the task"
    );
```